templates/Web/Pages/Vue.List/src/components/ListForm.vue

```diff
@@ -1,25 +1,59 @@
 ﻿<template>
   <form class="input-group my-3" @submit.prevent="$emit('onAddListItem')">
-    <input
-      :value="value"
-      aria-label="Add text here..."
-      class="form-control"
-      placeholder="Add text here..."
-      type="text"
-      @input="$emit('input', $event.target.value)"
-    />
-    <button type="submit" class="btn btn-primary ml-2">Submit</button>
+    <div class="col-11">
+      <input
+        :value="value"
+        aria-label="Add text here..."
+        class="form-control"
+        placeholder="Add text here..."
+        type="text"
+        @input="$emit('input', $event.target.value)"
+      />
+      <b-alert
+        variant="danger"
+        :show="!this.isValid"
+        dismissible
+      >Input must be at least 1 character long.</b-alert>
```

Comment:
Do you think it would be good to change the constant message and use the const here?

Reply:
I did the same thing as angular, look at #764